doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -525,3 +525,17 @@ It can only be included in one of the following Dataset properties:
 
 *   SPINEL_PROP_THREAD_MGMT_ACTIVE_DATASET
 *   SPINEL_PROP_THREAD_MGMT_PENDING_DATASET
+
+### PROP 5409: SPINEL_PROP_THREAD_CHILD_TABLE_ADDRESSES (#prop-thread-child-table-addresses)
+
+* Type: Read-Only
+* Packing-Encoding: `A(t(ESA(6))`
+
+This property provides the list of all addresses associated with every child
+including any registered IPv6 addresses.
+
+Data per item is:
+
+* `E`: Extended/long address of the child
```

Comment:
nit: let's try to avoid using multiple terms whenever possible.  The Thread and IEEE 802.15.4 specifications consistently use "extended", so let's drop "long".

Reply:
Good point. It's used in a couple of other places like this. I change the ones in spinel.h with this PR.